aioesphomeapi/api.proto

```diff
@@ -1297,13 +1297,25 @@ enum MediaPlayerState {
   MEDIA_PLAYER_STATE_IDLE = 1;
   MEDIA_PLAYER_STATE_PLAYING = 2;
   MEDIA_PLAYER_STATE_PAUSED = 3;
+  MEDIA_PLAYER_STATE_OFF = 4;
+  MEDIA_PLAYER_STATE_ON = 5;
```

Comment:
```suggestion
  # MEDIA_PLAYER_STATE_ANNOUNCING omitted because HA does not use it
  MEDIA_PLAYER_STATE_OFF = 5;
  MEDIA_PLAYER_STATE_ON = 6;
```

Reply:
We can leave it in.  The id can never be reused anyways so we are stuck with it forever even if HA doesn't use it.